File: hsqldb/jython/delete/hsqldb_delete.py
#! /usr/bin/jython
# -*- coding: utf-8 -*-
#
#	hsqldb_delete.py
#					Jun/02/2011
#
# ----------------------------------------------------------------
#
import sys
import	string
from java.lang import	System
#
import java
from java.sql import DriverManager
#
# ----------------------------------------------------------------
sys.path.append ('/var/www/data_base/common/jython_common')
from jython_rdb_manipulate import display_proc,delete_proc
# ----------------------------------------------------------------
System.out.println ("*** start ***")
print ("*** 開始 ***")
id_in = sys.argv[1]
print ("%s" % id_in)
#
java.lang.Class.forName("org.hsqldb.jdbcDriver")

#
database = "file:/var/tmp/hsqldb/cities;shutdown=true"
url="jdbc:hsqldb:" + database
user = "SA"
password = ""

conn = DriverManager.getConnection (url,user, password)
delete_proc	(conn,id_in)
display_proc (conn)
#
conn.close()
print ("*** 終了 ***")
#
# ----------------------------------------------------------------

File: hsqldb/jruby/delete/hsqldb_delete.rb
#! /usr/bin/jruby
# -*- coding: utf-8 -*-
#
#	hsqldb_delete.rb
#
#				Jan/29/2015
# ------------------------------------------------------------
#
include Java
import java.sql.DriverManager
import java.lang.System
#
#
load '/var/www/data_base/common/jruby_common/jruby_sql_manipulate.rb'
#
# ------------------------------------------------------------
puts "*** 開始 ***"
#
id_in = ARGV[0]
#
puts id_in
#
str_connect = "jdbc:hsqldb:file:/var/tmp/hsqldb/cities;shutdown=true"
#
conn = DriverManager.getConnection(str_connect,"SA", "")

sql_delete_proc(conn,id_in)
sql_read_proc(conn)

conn.close()
#
puts "*** 終了 ***"
#
# ------------------------------------------------------------
